src/core/EdFi.DataManagementService.Core.Tests.Unit/Middleware/ExtractDocumentInfoMiddlewareTests.cs

```diff
@@ -60,28 +61,22 @@ public async Task Setup()
                 )!
             );
 
+            string body = """{"schoolId": "123"}""";
+
             context = new(
                 new(
-                    Body: """{"schoolId": "123"}"""!,
+                    Body: body,
                     QueryParameters: [],
                     Path: "/ed-fi/schools",
                     TraceId: new TraceId("123")
                 ),
                 RequestMethod.POST
             )
             {
-                ResourceSchema = resourceSchema
+                ResourceSchema = resourceSchema,
+                ParsedBody = JsonNode.Parse(body)!
             };
 
-            if (context.FrontendRequest.Body != null)
```

Comment:
I made this much simpler. You are the owner of the setup, no need to do anything conditionally.